project.clj

```diff
@@ -79,7 +79,7 @@
                  [com.novemberain/pantomime "2.1.0"]
                  [fast-zip-visit "1.0.2"]
                  [robert/hooke "1.3.0"]
-                 [honeysql "0.5.2"]
+                 [honeysql "0.6.2"]
```

Comment:
I originally updated `honeysql` for the `array` function added in `0.5.3` but I didn't need it. Doesn't hurt to leave the upgrade though.